### PR TITLE
Menerjemahkan halaman tutorial index

### PR DESCRIPTION
**Pull request by iamyuu**
Ada beberapa kata yang tidak saya terjemahkan agar tetap sesuai.

### CODE REVIEW

File: docs/tutorial/index.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
 ---
-title: Gatsby.js Tutorials
+title: Tutorial Gatsby.js
 disableTableOfContents: true
 ---
 
-Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site.
+Selamat datang di Gatsby! Kami senang Anda ada di sini. Tujuan dari tutorial ini adalah untuk memandu Anda mengatur dan menyebarkan situs Gatsby Anda menggunakan templat pemula. Saat kita menjalani proses itu, kami akan memperkenalkan beberapa topik pengembangan web yang lebih umum, dan membahas struktur dasar situs Gatsby.
```
Comment (coroo) [inline docs/tutorial/index.md]: its better use: "Untuk memandu Anda dalam mengatur dan menyebarkan situs Gatsby pertama Anda"

Comment (coroo) [inline docs/tutorial/index.md]: Also, please change
"Saat kita menjalani proses itu,"
with 
"Dalam menjalani proses tersebut,"

File: docs/tutorial/index.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
 
-> The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/quick-start/) page.
+> Tutorial lengkap ini dimaksudkan agar dapat diakses oleh orang-orang tanpa banyak pengalaman pengembangan web (belum!) — tidak perlu menjadi seorang ahli. Jika Anda lebih suka melompat langsung ke kode, silakan lewati tutorial langkah demi langkah dan lihat halaman [mulai cepat](/docs/quick-start/).
```
Comment (coroo) [inline docs/tutorial/index.md]: Tutorial lengkap ini dimaksudkan agar dapat diakses oleh orang-orang yang belum memiliki banyak pengalaman dalam pengembangan web tidak perlu menjadi seorang ahli.

File: docs/tutorial/index.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
-1.  [Get to know Gatsby building blocks](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
-1.  [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
-1.  [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
+1.  [Menyiapkan lingkungan pengembangan Anda](/tutorial/part-zero/): Kami akan memperkenalkan Anda inti dari teknologi yang memberi kekuatan pada Gatsby, dan memandu Anda menyiapkan lingkungan pengembangan Anda.
+2.  [Mengenal blok bangunan Gatsby](/tutorial/part-one/): Memulai proyek baru, mengembangkan, dan menyebarkan situs.
+3.  [Pengantar menggunakan CSS di Gatsby](/tutorial/part-two/): Menjelajahi pustaka seperti Typography.js dan CSS Modules.
+4.  [Membangun tata letak bersarang di Gatsby](/tutorial/part-three/): Tata letak adalah bagian dari situs Anda yang digunakan kembali di beberapa halaman seperti header dan footer.
```
Comment (coroo) [inline docs/tutorial/index.md]: Its better to keep "nested layouts" with italic text. So the text will be : "Membangun _nested layouts_ di Gatsby"

File: docs/tutorial/index.md [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
 
-## Gatsby fundamentals
+## Dasar Gatsby
 
-0.  [Set up your development environment](/tutorial/part-zero/): We'll introduce you to core technologies that power Gatsby, and guide you through setting up your development environment.
-1.  [Get to know Gatsby building blocks](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
-1.  [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
-1.  [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
+1.  [Menyiapkan lingkungan pengembangan Anda](/tutorial/part-zero/): Kami akan memperkenalkan Anda inti dari teknologi yang memberi kekuatan pada Gatsby, dan memandu Anda menyiapkan lingkungan pengembangan Anda.
+2.  [Mengenal blok bangunan Gatsby](/tutorial/part-one/): Memulai proyek baru, mengembangkan, dan menyebarkan situs.
```
Comment (coroo) [inline docs/tutorial/index.md]: Keep it "building blocks", (with italic text) so it should be "Mengenal _building blocks_ pada Gatsby"

**Review by coroo — CHANGES_REQUESTED** [review]
I added some comment, please fix those first. I have another comment, but hopefully you can read it first before I added another comment again. Please, after you translate it, make sure you read it one more time again. And if you think you can't read it, please correct it first before push it. Thank you.

**Comment by iamyuu** [comment]
I fixed it on d858c8a, review please @coroo

**Review by coroo — APPROVED** [review]
Thank you @iamyuu 😄

**Pull request merged**